delivery.yaml

```diff
@@ -0,0 +1,30 @@
+build_meta:
```

Comment:
what about `version: "2017-09-20"`?

Reply:
This is still using the old format (since I just copied most of it from cdp's delivery.yaml 😳)